README.md

```diff
@@ -237,8 +237,8 @@ ad_defaults:
   active: true
   type: OFFER # one of: OFFER, WANTED
   description:
-    prefix: ""
```

Comment:
I would actually leave it as-is to not break existing configs, also because `description.prefix` is explicit enough.

Reply:
Alternatively allow defining `description_prefix` and `description_suffix` on top-level of the defaults config file to be consistent with the ad files config.